fdbclient/SystemData.h

```diff
@@ -76,6 +83,10 @@ const Value storageCacheValue( const std::vector<uint16_t>& serverIndices );
 void decodeStorageCacheValue( const ValueRef& value, std::vector<uint16_t>& serverIndices );
 
 //    "\xff/serverKeys/[[serverID]]/[[begin]]" := "" | "1" | "2"
+//	An internal mapping of what shards any given server currently has ownership of
+//	Using the serverID as a prefix, then followed by the beginning of the shard range
+//	as the key, the value indicates whether the shard does or does not exist on the server.
```

Comment:
what does value "2" mean? 
L85 indicates the value has three possible values, but commends only explain two choices.

Reply:
Actually, as far as I know, this is probably an outdated construct. There is no evidence of the value 2 ever being assigned to these special keys. The codebase only seems to use `serverKeysTrue` and `serverKeysFalse`, which are "1" and "", respectively. I can update L85 here or I can add more to the description if "2" is actually being used somewhere

Reply:
IIRC, I don't see "2" is used anywhere. Maybe delete "2" in L85 after double-checking with @sfc-gh-etschannen ?